src/mono/nuget/Microsoft.NET.Runtime.WebAssembly.Sdk/Sdk/AutoImport.props

```diff
@@ -0,0 +1,5 @@
+<Project>
+  <PropertyGroup>
+    <WasmNativeWorkload>true</WasmNativeWorkload>
```

Comment:
nit: is it about native only? are we going to distribute other parts differently?

Reply:
I'm fine with another name that indicates this pack is installed, I was using native to mean the native runtime build in this case

Reply:
I'd like to land _anything_ now so that it has a chance to flow though